include/aikido/robot/ConcreteManipulator.hpp

```diff
@@ -16,6 +16,11 @@ AIKIDO_DECLARE_POINTERS(ConcreteManipulator)
 class ConcreteManipulator : public Manipulator
 {
 public:
+  /// Expose base class functions
```

Comment:
[It seems Doxygen doesn't support description for the `using` declaration](https://stackoverflow.com/questions/36421785/doxygen-c-document-privately-inherited-members-exposed-by-the-using-declar). Let's make this non-doxygen comment (i.e., `// Expose base class functions`).